region/brinstar/pink.json

```diff
@@ -2814,12 +2814,13 @@
                   ]
                 },
                 {
-                  "name": "Waterway Suitless Spark",
+                  "name": "Suitless Spark",
                   "notable": false,
                   "requires": [
                     {"or": [
                       "SpaceJump",
-                      "canWalljump"
+                      "canWalljump",
+                      "canBombJumpWaterEscape"
```

Comment:
spring ball midair + careful
spring ball midair + hijump
~~and maybe on a separate strat: 
Super the caterpillar at the far left before it goes off screen, then freeze it on the wall~~